RadixWallet/Clients/OnLedgerEntitiesClient/OnLedgerEntitiesClient+Interface.swift

```diff
@@ -415,6 +415,31 @@ extension OnLedgerEntitiesClient {
 		return true // It is a pool unit resource address
 	}
 
+	public func isStakeClaimNFT(_ resource: OnLedgerEntity.Resource) async -> OnLedgerEntity.Validator? {
```

Comment:
This is exactly what I did for LSU, I return an optional Validator. So it must be good.